reference/7.3/Microsoft.PowerShell.Core/About/about_Telemetry.md

```diff
@@ -14,34 +14,45 @@ Describes the telemetry collected in PowerShell and how to opt-out.
 
 ## Long description
 
-PowerShell sends basic telemetry data to Microsoft. This data allows us to
-better understand the environments where PowerShell is used and enables us to
-prioritize new features and fixes. The telemetry is includes anonymized
-information about the host running PowerShell, and information about how
-PowerShell is used.
+PowerShell sends basic telemetry data to Microsoft using Application Insights.
+This data allows us to better understand the environments where PowerShell is
+used and enables us to prioritize new features and fixes. The telemetry is
+includes anonymized information about the host running PowerShell, and
+information about how PowerShell is used.
 
-The host-based telemetry data includes:
+PowerShell send the following information at startup:
 
-- OS name
-- OS version
-- PowerShell version
-- The PowerShell distribution channel
+- The name and version of the operating system
+- The version of PowerShell
+- The value of the **POWERSHELL_DISTRIBUTION_CHANNEL** environment variable
 - The version of the Application Insights SDK used by PowerShell
 - The geographic location of the host, based on the IP address
+- The parameters passed to pwsh without the parameter values
+- The Execution Policy setting of the current session
+- A randomly generated GUID representing the user running the instance
+- A randomly generated GUID representing the session instance
+
+Startup telemetry data is only collected when starting the `pwsh` executable.
+This information isn't sent if PowerShell is embedded in some other host
+application.
 
 > [!NOTE]
-> Application Insights uses the hosts IP address to determine the geographic
-> location. The IP address is never included in the telemetry data or stored
+> Application Insights uses the hosts IP Address to determine the geographic
+> location. The IP Address is never included in the telemetry data or stored
 > in the database.
 
-The following PowerShell activities are recorded:
+PowerShell sends the following information during the session:
+
+- The count of calls to the PowerShell.Create() API
+- The names and versions of Microsoft-owned modules loaded
```

Comment:
Do we need to specific which modules? Its just a whitelist https://github.com/PowerShell/PowerShell/blob/8d25b5ae6e2eb83a957eac823c27546a70c229a0/src/System.Management.Automation/utils/Telemetry.cs#L157

Reply:
I don't think so. I don't want to link to source code or document the list. We can point people to the source if they ask.